docs/src/lighting.md

```diff
@@ -0,0 +1,29 @@
+# Lighting
+
+For 3-D scenes, GLMakie offers several attributes to control the lighting of the scene.  These are set per Scene.
```

Comment:
They should be per plot atm. 

At some point we should split up these attributes into "Light" and "Material". Light (position, color) makes sense per-scene, Material (ambient, diffuse, specular (factor), shininess) per object.